portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1979,6 +2273,59 @@ definition {
 		}
 	}
 
+	@description = "LPS-124960 When there's a WebContent1 of type Subtype1 and WebContent2 of type Subtype2, and the user filters by Subtype1 and Subtype2, both web content are shown in the list"
+	@priority = "3"
+	test FilterBySubtypeFromDifferentAssets {
```

Comment:
FilterByType

Reply:
In this case, we already have a test called "FilterByType". The difference between them if that one filters only one type, and the one being developed filters multiple types. My suggestion is to name it "FilterByMultipleTypes"